### PR TITLE
Added get-connector-runs

### PR DESCRIPTION
**Pull request by rod-castillo**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [X] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
N/A

## Description
Added a new command to the KennaV2 integration that will get information on the recent runs of a specified connector

## Minimum version of Demisto
- [X] 5.0.0
- [ ] 5.5.0
- [ ] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [X] No

## Must have
- [X] Tests
- [X] Documentation

### CODE REVIEW

File: Packs/Kenna/Integrations/KennaV2/KennaV2.yml [hunk 1]
```diff
@@ -194,7 +184,10 @@ script:
       description: The status of the vulnerability. Can be "open", "closed", "risk_accepted",
         or "false_positive".
       isArray: true
-      name: status
+    - name: min-score
+      description: The minimum vulnerability score for which to return vulnerabilities.
+    - name: status
```
Comment (barchen1) [inline Packs/Kenna/Integrations/KennaV2/KennaV2.yml:189]: Looks like the command already has this argument. 
line 179

File: Packs/Kenna/Integrations/KennaV2/KennaV2.yml [hunk 2]
```diff
@@ -153,20 +143,20 @@ script:
...
     deprecated: false
     description: Executes a run of the specified connector. If file based, it will
       use the most recently uploaded data file.
     execution: true
-    name: kenna-run-connector
-  - arguments:
-    - default: false
+  - name: kenna-search-fixes
+    arguments:
+    - name: id
```
Comment (barchen1) [inline Packs/Kenna/Integrations/KennaV2/KennaV2.yml:159]: This command already exists.

File: Packs/Kenna/Integrations/KennaV2/KennaV2.yml [hunk 3]
```diff
@@ -153,20 +143,20 @@ script:
...
       type: Number
-  - arguments:
-    - default: false
-      description: The connector ID to run.
-      isArray: false
-      name: id
+    description: Searches for vulnerabilities in Kenna.
+  - name: kenna-run-connector
+    arguments:
+    - name: id
```
Comment (barchen1) [inline Packs/Kenna/Integrations/KennaV2/KennaV2.yml:149]: Missing argument description

File: Packs/Kenna/Integrations/KennaV2/KennaV2.yml [hunk 4]
```diff
@@ -271,28 +264,20 @@ script:
...
-    execution: false
-    name: kenna-update-asset
-  - arguments:
-    - default: false
-      description: The ID of the vulnerability to update.
-      isArray: false
-      name: id
+  - name: kenna-update-vulnerability
+    arguments:
+    - name: id
```
Comment (barchen1) [inline Packs/Kenna/Integrations/KennaV2/KennaV2.yml:280]: Missing argument description

**Review by barchen1 — CHANGES_REQUESTED** [review]
Thanks a lot for your contribution! You did a great job. 

Here are some notes about your PR:
1. Please add `1_1_0.md`release-note file using `demisto-sdk update-release-notes -i` https://xsoar.pan.dev/docs/integrations/release-notes
2. Missing lots of context output and human-readable for commands in the README file, please use `demisto-sdk generate-docs` https://xsoar.pan.dev/docs/integrations/integration-docs#run-demisto-sdk-generate-docs
3. Looks like you have duplicate data in your yml file, please go over the yml file and make sure there are no duplicates

Let me know if you have any questions

File: Packs/Kenna/Integrations/KennaV2/KennaV2.py [hunk 5]
```diff
@@ -230,14 +315,14 @@ def search_fixes(client: Client, args: dict) -> Tuple[str, Dict[str, Any], List[
     to_context = args.get('to_context')
     context: Dict[str, Any] = {}
     params = {
-        'id[]': argToList(args.get('id')),
-        'top_priority[]': argToList(args.get('top-priority')),
+        'id': argToList(args.get('id')),
+        'top_priority': argToList(args.get('top-priority')),
```
Comment (barchen1) [inline Packs/Kenna/Integrations/KennaV2/KennaV2.py:319]: Can you please explain why you changed these lines?

File: Packs/Kenna/Integrations/KennaV2/KennaV2.py [hunk 6]
```diff
@@ -339,10 +424,10 @@ def search_assets(client: Client, args: dict) -> Tuple[str, Dict[str, Any], List
     else:
         tags = args.get('tags')
     params = {
-        'id[]': argToList(args.get('id')),
-        'hostname[]': argToList(args.get('hostname')),
+        'id': argToList(args.get('id')),
+        'hostname': argToList(args.get('hostname')),
         'min_risk_meter_score': args.get('min-score'),
-        'tags[]': tags
+        'tags': tags
```
Comment (barchen1) [inline Packs/Kenna/Integrations/KennaV2/KennaV2.py:430]: Can you please explain why you changed these lines?

**Review by barchen1 — CHANGES_REQUESTED** [review]
I fixed most of the issues we had in this PR.
Please see the comments I gave you and answer them,
also, I added the `get-connector-runs` command to our current test-playbook `Packs/Kenna/TestPlaybooks/playbook-Kenna_Test.yml`, please take a look at the playbook and update me if we can delete the `Packs/Kenna/TestPlaybooks/TestKennaV2Playbook.yml` playbook.

Thanks

File: Packs/Kenna/Integrations/KennaV2/KennaV2.py (hunk 6 above)
Comment (rod-castillo) [reply Packs/Kenna/Integrations/KennaV2/KennaV2.py:430]: Hey Bar, I may have been working from an old copy of the integration, when I had started coding.  Not really sure which version of the line is correct.  It was not something that I had included in my changes.

File: Packs/Kenna/Integrations/KennaV2/KennaV2.yml [hunk 7]
```diff
@@ -514,7 +514,74 @@ script:
...
+      description: Connector Run End Time
+      type: string
+    - contextPath: Kenna.ConnectorRunsList.Success
+      description: Boolean value showing connector success
+      type: boolean
+    - contextPath: Kenna.ConnectorRunsList.TotalPayload
+      description: Total connector payloads
+      type: Number
+    - contextPath: Kenna.ConnectorRunsList.ProcessedPayload
+      description: total payloads processed by connector
```
Comment (guykeller) [inline Packs/Kenna/Integrations/KennaV2/KennaV2.yml:545]: @barchen1 Let's change to capital T

**Pull request closed without merging**